docs/integrations/sources/airbyte.md

```diff
@@ -0,0 +1,28 @@
+# Airbyte
+
+
```

Comment:
This source allows you to sync up data about your Airbyte Cloud workspaces. [Take a look at this guide](https://docs.airbyte.com/using-airbyte/configuring-api-access) to setup API access tokens.